app/src/main/java/com/example/sharingang/SearchFragment.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.sharingang

```

Comment:
We'll review this later when it's finished

Reply:
Remove it from the branch then?